### ISSUE TITLE
Update documentation throughout repo

### ISSUE THREAD

**Issue by spwoodcock** [hotosm/fmtm#921]
Replacement of https://github.com/hotosm/fmtm/issues/164

We currently have some minimal documentation, which could be greatly improved.
We would really welcome PRs for this 🙏 

## What needs to be done:

- Proof reading of current docs and correction.
- Reduce docs duplication - we currently have the same content duplicated in a few places.
- Improve general README, contributors guidelines, etc.
- Improve developer documentation with additional examples, FAQs (if you struggled setting up FMTM anywhere, feel free to document it!).
- Improve user documentation for how to use FMTM.
- Python docstrings throughout the backend, where missing (especially on routes).
- If you particularly like writing JSDoc docstrings, then it wouldn't harm having a few in the frontend too.

**Comment by vinayakhegde1** [comment]
@spwoodcock I've imrpoved the contributing guidelines please do check

**Comment by spwoodcock** [comment]
Reopening as q more general issue for contributors to help on.

It could definitely be split into sub issues based on the description though, to make it clearer what needs to be done

**Comment by manjitapandey** [comment]
Agreed, Sorry I mistakely closed it adding fixes and this issue. Not intended.

**Comment by Joshua-Tihabwangye** [comment]
hello, can i take on this isuue and complete it?

**Comment by manjitapandey** [comment]
Sure, Is there anything specific in your mind that you are thinking to add? 

Otherwise, I was thinking you could also add few video tutorials on mapper's page as well. or the product tool video?

**Comment by Joshua-Tihabwangye** [comment]
![Image](https://github.com/user-attachments/assets/0c519127-6494-4228-ba8a-13f41fb40f2e)

1.    I have read on the Field Mapper (The person who collects data)  section and I think  we should add in all the necessary apps by documenting there names and also providing the Links to those apps to enable the new or first time mappers to easily know the apps they are supposed to install which are compatible or which work hand in hand with FMTM.
2.  For easy access and to make the FMTM easy to work with also a link to the OSM registration site is needed below that step.

**Comment by Joshua-Tihabwangye** [comment]
![Image](https://github.com/user-attachments/assets/f2cffa90-1c0c-4358-aba1-014c620e7e8b)

**Comment by manjitapandey** [comment]
To do: 
1. Proof read the documentation for project manager and video manuals. 
2. Update with existing updates on workflow
3. Proofread if the user flow diagram is up to date or any section requires updating
4. Developer documentation updates if any part is outdated. 
5. Find out if there is any missing functionality that is yet to be documented.
6. If anything seems confusing or important, add those in FAQ section.

@Joshua-Tihabwangye

**Comment by Joshua-Tihabwangye** [comment]
Thank you @manjitapandey
